js/summary.go

```diff
@@ -21,21 +21,31 @@
 package js
 
 import (
-	"bytes"
+	_ "embed" // this is used to embed the contents of summary.js
 	"fmt"
 	"io"
+	"strings"
 	"time"
 
 	"github.com/dop251/goja"
 	"go.k6.io/k6/js/common"
 	"go.k6.io/k6/lib"
 	"go.k6.io/k6/stats"
-	"go.k6.io/k6/ui"
 )
 
-// TODO: move this to a separate JS file and use go.rice to embed it
-const summaryWrapperLambdaCode = `
+// Copied from https://github.com/k6io/jslib.k6.io/tree/master/lib/k6-summary
+//go:embed summary.js
+var jslibSummaryCode string //nolint:gochecknoglobals
+
+// TODO: move this or parts of it to a separate JS file and embed it like above
```

Comment:
:+1: but why not do this now? :) Reading JS in a Go file is cumbersome, which is fine for tests and small pieces of code, but this would be better in a standalone JS file.

Reply:
:+1: yeah, no reason not to do it now, will do so

Reply:
Resolved by https://github.com/k6io/k6/pull/1975/commits/6f6bfac43f69123c697c51740bf24899199609e8